simulator/jbuild

```diff
@@ -0,0 +1,26 @@
+(jbuild_version 1)
+
+(executable (
+  (name xenops_simulator_main)
+  (public_name xenopsd-simulator)
+  (package xapi-xenopsd-simulator)
```

Comment:
This wasn't installed either in the `_oasis` file, but if we want to install it now, that's fine.

Reply:
I thought we might want to, afer all it's installed on the server.

Reply:
Sounds good  - we've discussed that it's a good idea to keep the opam & XenServer installation in sync